qiskit_dynamics/solvers/diffrax_solver.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+# pylint: disable=invalid-name
+
+"""
+Wrapper for diffrax solvers
+"""
+
+from typing import Callable, Optional, Union, Tuple, List
+from scipy.integrate._ivp.ivp import OdeResult
+
+from qiskit_dynamics.dispatch import requires_backend
+from qiskit_dynamics.array import Array, wrap
+
+from .solver_utils import merge_t_args, trim_t_results
+
+try:
+    from diffrax import ODETerm, PIDController, SaveAt
+    from diffrax import diffeqsolve as _diffeqsolve
+
+    from diffrax.solver import AbstractSolver
+    import jax.numpy as jnp
+except ImportError as err:
+    pass
+
+
+@requires_backend("jax")
+def diffrax_solver(
+    rhs: Callable,
+    t_span: Array,
+    y0: Array,
+    method: "AbstractSolver",
+    t_eval: Optional[Union[Tuple, List, Array]] = None,
+    **kwargs,
+):
+    """Routine for calling `diffrax.diffeqsolve`
```

Comment:
Make sure to put two ticks here:
```
``diffrax.diffeqsolve``
```
this doesn't get rendered in the docs anyway but might as well use correct sphinx syntax